src/main/java/frc/robot/subsystems/LimelightInterface.java

```diff
@@ -15,87 +15,33 @@ public class LimelightInterface extends SubsystemBase{
 
     private static final NetworkTable limelight = NetworkTableInstance.getDefault().getTable("limelight");
 
-
-
     private static boolean tag = false;
```

Comment:
suggestion: name the variable -> tagDetected, tagVisible or something similar so is clear what true means